bibliopixel/remote/trigger_process.py

```diff
@@ -0,0 +1,10 @@
+from .. util import importer
```

Comment:
Hah, madness!  :-D 

The process is fine.  I just wanted to make sure you were sure.